egs/swbd/s5c/local/chain/tuning/run_blstm_6i.sh

```diff
@@ -130,6 +130,7 @@ if [ $stage -le 12 ]; then
     --recurrent-projection-dim 256 \
     --non-recurrent-projection-dim 256 \
     --label-delay $label_delay \
+    --self-repair-scale-clipgradient 1.0 \
```

Comment:
1.0 is the default value set for self-repair-scale-clipgradient in make_configs.py. Here I just explicitly tell the user there is such option.